### ISSUE TITLE
Laravel scout v9 support

### ISSUE THREAD

**Issue by guvenakcoban** [yabhq/laravel-scout-mysql-driver#120]
https://github.com/laravel/scout/releases/tag/v9.0.0

Does it affect this package? If not, can you please update composer.json

**Comment by erfantkerfan** [comment]
please fix this
I'm the 3rd person who gave thumbs up to this issue
@toonevdb

**Comment by nogamenofun98** [comment]
I also wondering is it working with laravel scount v9.

**Comment by dherran** [comment]
Support for Scout v9 was merged into master yesterday. Just waiting on a release.

**Comment by ghost** [comment]
A release needs to be tagged for L9 support please.

[EDIT] Actually there is no Laravel v9 support yet, just Scout v9.
